client/package-lock_LOCAL_232123.json

```diff
@@ -0,0 +1,10810 @@
+{
```

Comment:
I asked you to delete these kind of files that are generating by git during merge conflicts